tests/SConscript

```diff
@@ -1,3 +1,7 @@
 # -*- python -*-
 from lsst.sconsUtils import scripts
-scripts.BasicSConscript.tests()
+
+with open('test.txt', 'r') as f:
+    tests = f.readlines()
+pybind11_ported_tests = [t[6:].strip() for t in tests if not t.startswith('#')]
+scripts.BasicSConscript.tests(pyList=pybind11_ported_tests)
```

Comment:
If all tests work, just put it back to normal and remove `test.txt`. That is only meant to make sure that rebuild / Jenkins works while not all tests have been wrapped.